pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -216,6 +216,8 @@ spec:
           #we can't use array parameters directly here
           #we pass them in as goals
           /usr/bin/mvn -s "$(workspaces.maven-settings.path)/settings.xml" $@ "-DaltDeploymentRepository=local::file:$(workspaces.source.path)/hacbs-jvm-deployment-repo" "org.apache.maven.plugins:maven-deploy-plugin:3.0.0-M2:deploy" || { cat $(workspaces.maven-settings.path)/sidecar.log ; false ; }
+          #TODO conversely, since deploy-and-check-for-contaminates is catting the sidecar log regardless, should we do that here?
+          cat $(workspaces.maven-settings.path)/sidecar.log
```

Comment:
@stuartwdouglas so yeah whould we add this?

Reply:
I don't think we need it. It will get printed by deploy-and-check-for-contaminates regardless. If this step is failing then the version on the line above will print it.

Reply:
yeah I could only think of some amount of "intemediate state" in the "successful logs" in this step vs. "failing logs" in the subsequent step, where it would be easier to determin the delta by having both preent

admittedly a most likely edge sort of case ... I'll remove the commit

Reply:
You should be able to check the timestamps to figure out the difference anyway.